prime-router/src/testIntegration/kotlin/datatests/mappinginventory/catchall/hd/HDToOrganizationTests.kt

```diff
@@ -42,7 +42,7 @@ class HDToOrganizationTests {
     }
 
     @Test
-    fun `test that Organization country uses MSH17 when MSH17 present`() {
```

Comment:
I updated this test scenario to populate MSH.4 and MSH.17 in the HL7 to better display a fully populated HD/Organization.